emcc.py

```diff
@@ -389,7 +389,7 @@ def run():
 
   if DEBUG:
     logging.warning('invocation: ' + ' '.join(sys.argv) + (' + ' + EMCC_CFLAGS if EMCC_CFLAGS else '') + '  (in ' + os.getcwd() + ')')
-  if EMCC_CFLAGS:
+  if EMCC_CFLAGS and "-v" not in sys.argv:
```

Comment:
please add a comment explaining why `-v` is relevant here - it's not immediately obvious (without this PR's context) I don't think